pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1109,13 +1109,18 @@ protected void handleSubscribe(final CommandSubscribe subscribe) {
                 TopicOperation.CONSUME
         );
 
+        CompletableFuture<Boolean> isAuthorizedToCreateTopicFuture = isNamespaceOperationAllowed(
+                topicName.getNamespaceObject(),
+                NamespaceOperation.CREATE_TOPIC
+        );
```

Comment:
As I mentioned before, this does not look backwards/forwards compatible. I don't think we can add required `AuthActions` without finding a way to make them compatible. The idea is that users should be able to upgrade then downgrade and things should still work. Can you confirm @KannarFr?

Reply:
That would be nice, but as I answered you am I not sure we can really do it has CREATE_TOPIC check is already performed in HTTP admin checks.

Reply:
While I agree that the current design is inconsistent, I don't think we should add logic that breaks existing use cases.